assets/js/app/inventory/ItemDetail.jsx

```diff
@@ -399,13 +396,19 @@ const ItemDetail = React.createClass({
         var tagArray = tags;
       }
 
-      state.tags = tagArray;
 
       ajax({
         url:"/api/items/" + thisobj.state.item.name + "/",
         type: "PUT",
         traditional: true,
-        data: state,
+        data: {
```

Comment:
we should standardize how we send json. I think stringifying the whole thing is better